src/components/atoms/Checkbox.tsx

```diff
@@ -0,0 +1,21 @@
+import {useState} from 'react'
+
+type CheckboxProps = {
+  text: string
+  checked: boolean
+}
+
+export default function Checkbox({text, checked}: CheckboxProps) {
+  const [state, setState] = useState(false)
+  return (
+    <label htmlFor={text}>
+      <input
+        type="checkbox"
+        defaultChecked={checked}
+        onChange={() => setState(prev => !prev)}
```

Comment:
Можно использовать `event` и задавать `setState` исходя из него:
```suggestion
        onChange={(event) => setState(event.checked)}
```